### ISSUE TITLE
User ta cannot select secure storage

### ISSUE THREAD

**Issue by zhupg** [OP-TEE/optee_os#934]
It is possible to use the normal world fs and the RPMB simultaneously in the last version. But user ta cannot select which one to use, only can use TEE_STORAGE_PRIVATE.

In the file: lib/libutee/tee_api_objects.c
```c
if (storageID != TEE_STORAGE_PRIVATE) {
		res = TEE_ERROR_ITEM_NOT_FOUND;
		goto out;
}
```

There is parameter check, I think it should also check if it is TEE_STORAGE_PRIVATE_RPMB  or 
TEE_STORAGE_PRIVATE_REE, which are also correct.

**Comment by jforissier** [comment]
Thanks for reporting.
You're right, the tests in libutee are invalid. Would you want to submit a pull request? To avoid code duplication, I suggest simply removing the checks and let the TEE kernel validate the value instead. Also, it would be nice if `xtest` would run the storage tests on all the FS that are enabled at compile time (see `xtest_6000.c` ; there are other storage tests from the GP suite but they might be a bit harder to adjust I think).

**Comment by zhupg** [comment]
Dear jforissier:
I have submit a pull request: #938 
Would you please help review and merge? Thanks.

**Comment by zhupg** [comment]
Fixed in #938
